File: HomeWork05/Commands/ControlllerCommands/ExitAppCommand.java
package Commands.ControlllerCommands;
import Commands.ControlllerCommands.Base.ControllerCommand;
import Controllers.Controller;

public class ExitAppCommand extends ControllerCommand{
 
    public ExitAppCommand(Controller controller) {
        super(controller);
    }

    @Override
    public void execute() {
        controller.getScreen().setBar("До свидания.");
        controller.setExitApp();
    }

}

File: HomeWork07/Commands/ControlllerCommands/ExitAppCommand.cs
namespace HomeWork07;

public class ExitAppCommand : ControllerCommand
{

    public ExitAppCommand(Controller controller) : 
    base(controller)
    {
    }

    public override void execute()
    {
        controller.getScreen().setBar("До свидания.");
        controller.setExitApp();
    }
}
